pkgs/by-name/po/pom/package.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pom";
+  version = "unstable-2024-04-29";
```

https://github.com/NixOS/nixpkgs/blob/7001f7433563decbc59405bb3d7a95575543576c/pkgs/README.md

> - The `version` attribute _must_ start with a digit e.g., `"0.3.1rc2"`.
> 
> - If a package is a commit from a repository without a version assigned, then the `version` attribute _should_ be the latest upstream version preceding that commit, followed by `-unstable-` and the date of the (fetched) commit. The date _must_ be in `"YYYY-MM-DD"` format.
> 
> Example: Given a project had its latest releases `2.2` in November 2021, and `3.0` in January 2022, a commit authored on March 15, 2022 for an upcoming bugfix release `2.2.1` would have `version = "2.2-unstable-2022-03-15"`.
> 
> - If a project has no suitable preceding releases - e.g., no versions at all, or an incompatible versioning / tagging schema - then the latest upstream version in the above schema should be `0`.
> 
> Example: Given a project that has no tags / released versions at all, or applies versionless tags like `latest` or `YYYY-MM-DD-Build`, a commit authored on March 15, 2022 would have `version = "0-unstable-2022-03-15"`.

```suggestion
  version = "0-unstable-2024-04-29";
```

---

Thank you!